pkg/webhook/external/artifactory.go

```diff
@@ -148,41 +149,51 @@ func (a *artifactoryWebhookReceiver) getHandler(requestBody []byte) http.Handler
 			return
 		}
 
-		originURL, err := url.Parse(payload.Origin)
-		if err != nil {
+		if payload.Data.ImageType != artifactoryDockerDomain && payload.Data.ImageType != artifactoryChartImageType {
```

Comment:
I would add the supported types to a `var` slice and use `slices.Contains` here.